dashboard/config/locales/scripts.en.yml

```diff
@@ -200,6 +206,7 @@ en:
           lessons:
             Artist:
               name: Artist
+          student_description: Let's use code to join Anna and Elsa as they explore the magic and beauty of ice. You will create snowflakes and patterns as you ice-skate and make a winter wonderland that you can then share with your friends!
```

Comment:
My understanding is that we'll want the added translations in this file to get merged in an earlier PR, then allow crowdin to populate translations, before merging the code changes in this PR. Otherwise we might have a period of time where student descriptions appeared in english when viewed in other languages. @bethanyaconnor or @Hamms is that correct?

Reply:
That would be ideal but if it would block other work, I think it's okay to merge this whole change earlier. These strings will have translations once next week's sync runs as they're duplicates, so the turn around is going to be pretty small.

That said, if anything holds up this PR, it would be really great to check in this file before Sunday.